---
# Documentation: https://wowchemy.com/docs/managing-content/

title: "rMAP: Rapid Microbial Analysis Pipeline for ESKAPE bacterial group whole-genome sequence data"
authors: 
- admin
- Gerald Mboowa

# Author notes (optional)
author_notes:
- "Equal contribution"
- "Equal contribution"

date: 2021-05-12T23:07:05+03:00
doi: ""

# Schedule page publish date (NOT publication's date).
publishDate: 2017-05-12T23:07:05+03:00

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: ""
publication_short: ""

abstract: "The recent re-emergence of multidrug resistant pathogens through persistent misuse antibiotics has
exacerbated their threat to worldwide human public health and well-being. The evolution of the
genomics era has led to generation of huge volumes of sequencing data at an unprecedented rate
due to the ever-reducing costs of whole-genome sequencing (WGS). The considerable bioinformatics
skills needed to analyze the large volume of genomic data from these platforms and subsequent
downstream analysis offer constraints in the implementation of WGS as a routine laboratory
technique. We have developed rMAP, a user-friendly pipeline capable of profiling the resistomes of
ESKAPE pathogens (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species). rMAP is designed for beginners and people with little bioinformatics expertise, and automates the steps required for WGS analysis directly from the raw genomic sequence data including: adapter and low quality sequence read trimming, de-novo genome assembly, genome annotation, SNP-variant calling, phylogenetic inference by maximum-likelihood, antimicrobial resistance profiling, plasmid profiling, virulence factor determination, multi-locus
sequence typing (MLST), pangenome analysis, and insertion sequence characterization (IS)."

# Summary. An optional shortened abstract.
summary: ""

tags: []
categories: []
featured: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_pdf:
url_code:
url_dataset:
url_poster:
url_project:
url_slides:
url_source:
url_video:

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---
